doc/cygnus-twitter/user_and_programmer_guide/twitter_hdfs_sink.md

```diff
@@ -0,0 +1,284 @@
+# <a name="top"></a>TwitterHDFSSink
```

Comment:
I would add this file to `doc/cygnus-twitter/flume_extensions_catalogue`, specially if it is intented to have more Twitter-related sinks in the future.